app/scripts/components/modeldetails.js

```diff
@@ -90,7 +90,10 @@ var exports = (function () {
       delft3DVersion: {
         cache: false,
         get: function () {
-          return this.activeModel.data.versions.delft3d.delft3d_version;
+          if(_.has(this.activeModel, "data.versions.delft3d.delft3d_version")) {
+            return this.activeModel.data.versions.delft3d.delft3d_version;
```

Comment:
I get what you mean, but this will incorporate many changes - I will create a task for it in Jira.